### PR TITLE
Migrate to Gulp and ESLint. Import Bootstrap 4 and add Stylelint.

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

File: dist/images/visionapps.svg [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+				<g>
+					<rect x="60.211" y="18.904" fill="#666666" width="9.266" height="52.344"/>
+				</g>
+				<path fill="#666666" d="M0,18.904h9.97l15.005,41.027l14.906-41.027h9.97L28.565,71.247h-8.574L0,18.904z"/>
+				<path fill="#666666" d="M307.502,11.65l-23.523,59.598h-11.503L302.719,0h10.214l30.393,71.248h-12.151L307.502,11.65z"/>
+			</g>
+		</g>
+	</g>
+</g>
+<g id="Vrstva_2" display="none">
```
Comment (dlouhak) [inline dist/images/visionapps.svg]: Is this element necessary? I would delete it.

File: dist/images/visionapps.svg [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 17.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="507.314px" height="90.191px" viewBox="0 0 507.314 90.191" enable-background="new 0 0 507.314 90.191"
+	 xml:space="preserve">
+<g id="Vrstva_3">
+</g>
+<g id="Vrstva_1">
+	<g>
```
Comment (dlouhak) [inline dist/images/visionapps.svg]: Doesn't we indent 2 instead of 4 spaces in svg?

File: LICENSE.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014–2016 VisionApps, s.r.o.
+Copyright (c) 2014–2018 VisionApps, s.r.o.
```
Comment (mbohal) [inline LICENSE.txt]: As of 1st January 2019 this will be wrong. I would remove the years if possible.

File: gulpfile.js [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+  return del('dist/images');
+}
+
+function copyImages() {
+  return gulp
+    .src('src/images/**')
+    .pipe(gulp.dest('dist/images'));
+}
+
+function bs(done) {
```
Comment (mbohal) [inline gulpfile.js]: two letter function names are not too descriptive. Could we write it out in full?

File: LICENSE.txt (hunk 3 above)
Comment (adamkudrna) [reply LICENSE.txt]: In the US the copyright is not effective unless you put a year here (as far as I remember). It should stay here.

**Pull request merged**